CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/ViewControllers/HelpTipsViewController.swift

```diff
@@ -48,6 +49,9 @@ public final class HelpTipsViewController: UIViewController {
     }
 
     public func configureMainView() {
+        if UIDevice.current.userInterfaceIdiom == .pad {
+            horizontalMargin = 126 - 16
```

Comment:
@chriskryn can we use constants here something like:
private let marginX = 126
            horizontalMargin = marginX - margin